src/components/DataTable/data-table.scss

```diff
@@ -0,0 +1,246 @@
+.sb-data-table {
+  &__loading {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+
+  &__container {
+    width: 100%;
+    border-spacing: 0;
+    border-collapse: collapse;
+  }
+
+  &--loading {
+    position: relative;
+
+    .sb-loading {
+      position: absolute;
+      @include middleXY();
+    }
+
+    .sb-data-table__body-cell {
+      text-indent: -999em;
+      position: relative;
+
+      &:before {
+        content: '';
+        width: 64px;
+        height: 10px;
+        background-color: $light-50;
+        position: absolute;
+        left: 10px;
+        top: 50%;
+        transform: translateY(-50%);
+      }
+    }
+
+    .sb-data-table__col-selection:before,
+    .sb-input-checkbox {
+      opacity: 0;
+    }
+  }
+
+  &__row {
+    border-bottom: 1px solid $light-50;
+
+    &:first-child {
+      border-top: 1px solid $light-50;
+    }
+
+    &:nth-child(even) {
+      .sb-data-table--striped & {
+        background-color: $light-25;
+      }
+    }
+
+    &:hover {
+      background-color: $light-50 !important;
+      border-bottom: 1px solid $white;
+    }
+
+    &--selected {
+      background-color: $light-50;
+      border-bottom: 1px solid $white;
+
+      &:hover {
+        background-color: $light-75 !important;
+      }
+    }
+  }
+
+  &__head-cell, &__body-cell {
+    color: $light-gray;
+    font-size: 14px;
+    font-weight: normal;
+    text-align: left;
+    padding: 18px 10px;
+
+    &.sb-data-table__col-main {
+      font-weight: 500;
+      color: $sb-dark-blue;
+    }
+  }
+
+  &__head-cell {
+    &:hover {
+      .sb-data-table__sort-icon {
+        opacity: 1;
+      }
+    }
+  }
+
+  &__sort-icon {
+    margin-left: 10px;
+    opacity: 0;
+
+    .sb-data-table--show-icon & {
+      opacity: 1;
+    }
+  }
+}
+
+$checkbox-transition: .2s cubic-bezier(0,0,.2,1) 0ms;
+
+.sb-input-checkbox {
+  vertical-align: middle;
+  outline: 0;
+
+  &__bg {
+    cursor: pointer;
+    top: 50%;
+    left: 50%;
+    width: 50%;
+    height: 50%;
+    background-color: $color-white;
+    border: 1px solid $light;
+    border-radius: 3px;
+    transition: background, border $checkbox-transition;
+    position: absolute;
+    transform: translate(-50%, -50%);
+
+    &:hover {
+      border-color: $color-primary;
+    }
+  }
+
+  &__native {
```

Comment:
You could use a different hack to hidden the native input element, so this way hidden the input to screen readers too. Take a look on this link from [webaim](https://webaim.org/techniques/css/invisiblecontent/). But, in nutshell, you can use the following class helper:

```css
.sr-only {
  position:absolute;
  left:-10000px;
  top:auto;
  width:1px;
  height:1px;
  overflow:hidden;
}
```